### PR TITLE
[DOCS] Community handbook link in readme | CU-1ydvcdq

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
Adds community link in readme
minor height fix of img 

This PR fixes # CU-1ydvcdq

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
Task linked: [CU-1ydvcdq Spread around the link the Community Handbook](https://app.clickup.com/t/1ydvcdq)

**Comment by Chadha93** [comment]
@leecalcote is this the desired place to add this? if Yes I should proceed to add this across others too.

**Review by warunicorn19 — APPROVED** [review]
Merge at-will @Chadha93

**Comment by Chadha93** [comment]
@warunicorn19 Yass, just get one more approval, I'm always happy to hit merge 😂

**Comment by l5io** [comment]
🚀 Preview for commit 0b73b75fbc5f1d2d2ff63612b283e9d156c72529 at: https://61e5b192d5eabe94d3996afe--layer5ng.netlify.app

**Comment by Chadha93** [comment]
This also generates a preview when Readme update, nice!!

File: README.md [hunk 1]
```diff
@@ -128,12 +128,13 @@ alt="Layer5" align="left" /></a>
 Join us on<a href="http://slack.layer5.io"><img src=".github/assets/images/slack.png" height="16px" align="bottom" />Slack</a>to learn more about Layer5 and its community! Make sure you see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you.
 </p>
 <p>
-<a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src=".github/assets/images/buttons/community.png" style="margin-right:10px;" width="125px" align="left" /></a>
+<a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src=".github/assets/images/buttons/community.png" style="margin-right:10px;" width="125px" height="110px" align="left" /></a>
```
Comment (leecalcote) [inline README.md:131]: Is there a need to specify this? Specifying it has the affect of preventing resizing.

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**